File: src/main/java/com/arnas/klatrebackend/features/invites/InviteService.java
package com.arnas.klatrebackend.features.invites;

import java.util.List;

public interface InviteService {

    void sendInvite(long userId, long groupId, long invitedUserId);
    List<GroupInviteDisplay> getUserPendingInvites(long userId);
    void acceptInvite(long inviteId, long userId);
    void rejectInvite(long inviteId, long userId);
}

File: src/main/kotlin/com/arnas/klatrebackend/features/invites/InviteService.kt
package com.arnas.klatrebackend.features.invites

interface InviteService {
    fun sendInvite(userId: Long, groupId: Long, invitedUserId: Long)
    fun getUserPendingInvites(userId: Long): List<GroupInviteDisplay>
    fun acceptInvite(inviteId: Long, userId: Long)
    fun rejectInvite(inviteId: Long, userId: Long)
}

